### PR TITLE
[R20-892] Fix validation errors

### PR DESCRIPTION
**Pull request by waitingallday**
**Issue**: https://digital-vic.atlassian.net/browse/R20-892

### What I did

- Fixed markup and CSS errors to validate document, mostly uncaught `v-bind` props exposed on render, html hierarchy and some incorrect aria uses (underlying elements have changed in a few cases)
- Added required default media condition to img `sizes` that have multiple sets
- Fixed a CSS error upstream in `ol-ext`
- Removed unneeded `x-ua-compatible` meta from static html templates (support for Internet Explorer 11 ended on June 15, 2022 🎉  and edge doesn't support this, also nuxt doesn't render it either)
- Added a plugin to force prettier to correctly allow void elements (`link`, `meta`, etc.) to render without self-closing tags

Screenshot 2024-06-03 at 4 46 55 pm

### How to test

- https://validator.w3.org/nu/#textarea
- 

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

File: examples/nuxt-app/test/fixtures/case/multi-step-webform.json
Comment (jeffdowdle) [inline examples/nuxt-app/test/fixtures/case/multi-step-webform.json:1]: @waitingallday is this file here intentionally?

Comment (waitingallday) [reply examples/nuxt-app/test/fixtures/case/multi-step-webform.json:1]: Oops, no it's not

File: packages/ripple-ui-core/src/components/header/RplHeroHeader.vue [hunk 1]
```diff
@@ -120,6 +120,7 @@ const handleClick = (event) => {
         v-bind="background"
         priority="high"
         :aspect="backImageRatio"
+        :drupal_internal__target_id="null"
```
Comment (lambry) [inline packages/ripple-ui-core/src/components/header/RplHeroHeader.vue:123]: Can we avoid having a drupal specific prop on our core-ui component?

File: packages/ripple-ui-core/src/components/campaign-banner/RplPrimaryCampaign.vue [hunk 2]
```diff
@@ -52,6 +52,7 @@ const handleClick = () => {
         v-bind="image"
         :aspect="{ xs: 'wide', s: 'panorama' }"
         sizes="xs:715px"
+        :drupal_internal__target_id="null"
```
Comment (dylankelly) [inline packages/ripple-ui-core/src/components/campaign-banner/RplPrimaryCampaign.vue:55]: lets see if we can remove this in the mapping before it get to the component eg: https://github.com/dpc-sdp/ripple-framework/blob/4de1afcc218fd594273997c1fa1c824747b68e1f/packages/ripple-tide-api/src/utils/mapping-utils.ts#L57

File: packages/ripple-ui-core/src/components/header/RplHeroHeader.vue (hunk 1 above)
Comment (lambry) [reply packages/ripple-ui-core/src/components/header/RplHeroHeader.vue:123]: Marking as resolved, in favour of https://github.com/dpc-sdp/ripple-framework/pull/1202#discussion_r1625179487

**Review by dylankelly — APPROVED** [review]

**Review by jeffdowdle — APPROVED** [review]

**Pull request merged**